lib/perl/Genome/Model/Tools/Picard.pm

```diff
@@ -497,9 +481,11 @@ sub _parse_metrics_file_into_hashref {
     unless (defined $metric_header_as_key) {
         if ($class->can('_metric_header_as_key')) {
             $metric_header_as_key = $class->_metric_header_as_key;
+        } elsif ($accumulations) {
+            $metric_header_as_key = shift @$accumulations;
         } else {
-            $class->debug_message('Assuming the first column is the key for the metrics hashref in file: '. $metrics_file);
-            $metric_header_as_key = $reader->headers->[0];
+            #There's only one thing... or something is wrong!
```

Comment:
Instead of a comment, should we make this an assertion of some sort?

```perl
my $data = $reader->next;
die $class->error_message("Only expected one line but found multiple") if $reader->next;
return $data;
````